DESCRIPTION

```diff
@@ -25,6 +25,11 @@ Suggests:
     covr
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.0.2
+Depends:
+    R (>= 2.10.0)
```

Comment:
This has been changed in PR #28. Thus, has to remove these two lines after update.